.github/workflows/publish.yml

```diff
@@ -261,7 +263,7 @@ jobs:
         dotnet publish ArchiSteamFarm -c "$CONFIGURATION" -o "out/${VARIANT}" "-p:ASFVariant=${VARIANT}" -p:ContinuousIntegrationBuild=true --nologo $variantArgs
 
         # If we're including official plugins for this framework, copy them to output directory
-        for plugin in $PLUGINS; do
+        for plugin in $DEFAULT_PLUGINS; do
```

Comment:
Why this plugin is built in this workflow if it doesn’t get published anyway? You can use `ci.yml` if you want to add a build check

Reply:
It doesn't get published now, but the target is to have it be published as `MonitoringPlugin.zip` (not as part of mainline ASF, but a separate .zip file) and have it be automatically updated if added (plugin is derived from/implementing `IGitHubPluginUpdates`). I'm just not comfortable fully doing that with these pipelines because... Archi :P